bedrock/externalpages/templates/externalpages/pocket/add.html

```diff
@@ -1,9 +1,110 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
 {% extends "externalpages/pocket/base.html" %}
 
 {% block title %} How to Save {% endblock%}
 
 {% block content %}
 
-<h1>How to Save</h1>
+<div class="add-page">
+   <section class="add-header">
+      <div class="add-header__wrapper">
+         <div class="add-header__title">
+            <h2>Save to Pocket from Firefox</h2>
+         </div>
+         <div class="add-header__context">
+            <div class="add-header__context-image"><img src="{{ static('img/externalpages/pocket/add-pocket-header.jpg') }}" alt="Pocket logo in browser taskbar"></div>
+            <div class="add-header__context-info">
+               <p>
+                  Click the Pocket button in your Firefox toolbar to save articles, videos,
+                  pages, and more.
+               </p>
+               <a href="https://help.getpocket.com/article/900-saving-to-pocket-and-viewing-your-list-in-firefox" class="btn red-btn" target="_blank" rel="noopener noreferrer">Learn How</a>
+               <p>
+                  <a
+                     href="https://help.getpocket.com/article/942-where-is-the-pocket-button-in-firefox"
+                     class="dark-inline-link" target="_blank" rel="noopener noreferrer">
+                  Don't see the Pocket button?
+                  <span class="rarrow">›</span>
```

Comment:
Think we could replace this with the corresponding unicode character

Reply:
The unicode character looks quite clunky compared to this one, and looks quite different from what they have on the [website](https://getpocket.com/add)